classes/class-wc-connect-service-settings-store.php

```diff
@@ -206,7 +208,49 @@ public function get_enabled_services() {
 
 			usort( $enabled_services, array( $this, 'sort_services' ) );
 			return $enabled_services;
+		}
+
+		/**
+		 * Checks if the shipping method ids have been migrated to the "wc_services_*" format and migrates them
+		 */
+		public function migrate_legacy_services() {
+			if ( WC_Connect_Options::get_option( 'shipping_methods_migrated', false ) //check if the method have already been migrated
+				|| ! $this->service_schemas_store->fetch_service_schemas_from_connect_server() //ensure the latest schemas are fetched
+				|| ! $this->service_schemas_store->get_all_shipping_method_ids() ) { //verify that the schemas contain the method_id field
```

Comment:
This check isn't needed. We can safely release the server before the client, server's changes are backwards-compatible. It doesn't hurt though, so leave it if you want.